spec/linters/coala/coala_json.json

```diff
@@ -0,0 +1,678 @@
+{
+  "results": {
+    "default": [
+      {
+        "additional_info": "",
+        "affected_code": [
+          {
+            "end": {
+              "column": null,
+              "file": "/home/mauricio/test-coala/ReadImage.java",
```

Comment:
Yes, this is the output from coala json. It is not being used as a path to find files, just for testing propouses